simple/stats/importer.py

```diff
@@ -97,13 +99,17 @@ def _rename_columns(self) -> None:
     # Rename SV columns to their IDs
     sv_column_names = self.df.columns[2:]
     sv_ids = [
-        self.nodes.variable(sv_column_name).id
+        self.nodes.variable(sv_column_name, self.input_file_name).id
         for sv_column_name in sv_column_names
     ]
     renamed.update({col: id for col, id in zip(sv_column_names, sv_ids)})
 
     self.df = self.df.rename(columns=renamed)
 
+  def _add_provenance_column(self):
+    self.df[constants.COLUMN_PROVENANCE] = self.nodes.provenance(
+        self.input_file_name).id
```

Comment:
General/Broader point (nothing to do in this PR):  not sure how heavily dependent we are on pandas, but might be nice to not assume the entire CSV needs to be memory resident.  May allow the tool to scale to many GBs of CSV...

Reply:
Pandas is mostly here (triples also but that's minor). Given the programmatic convenience of pandas, one approach for scaling could be to partition large csvs into smaller chunks and process each separately with pandas. I worry about the coding complexity and maintenance costs associated with hand coding this since we are likely to continue to add features and support for other formats.

Reply:
Sure.   Though, in the abstract, Pandas makes is easy to do joins (do we?), but outside of that its often just a for-loop difference.  If we parallelize using dataflow (flume) that's another reason to process rows in code.

(I'm wary of pandas for anything core given the hidden costs, the most recent being the latency surprise.)

Anyway, just something to keep in mind...

Reply:
Ack.